test/Libraries/SystemTestServices/SystemTestBase.cs

```diff
@@ -305,15 +306,29 @@ public void AssertPreviewCount(string guid, int count)
             Assert.AreEqual(count, data.GetElements().ToList().Count);
         }
 
-        public NodeModel GetNode<T>(string guid) where T : NodeModel
+        /// <summary>
+        /// Returns the NodeModel corresponding to the input guid
+        /// </summary>
+        /// <typeparam name="T">A NodeModel (or derived) type</typeparam>
+        /// <param name="guid">The node Guid</param>
+        /// <returns></returns>
+        /// <exception cref="Exception"></exception>
+        [Obsolete("This method will be removed in a future version of Dynamo")]
```

Comment:
will it be replaced with some other method?

Reply:
Yes, I just did not want to overcomplicate de PR now.  I can change the text of the obsolete method wen the new one is implemented